.github/workflows/i18n-check.yml

```
@@ -0,0 +1,15 @@
+name: i18n Check
+
+on: [push, pull_request]
+
+jobs:
+  check-coverage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)